examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
And as far as picks goes - you're right that `make_projector_info` calls `make_projector` which does let you supply a list of channel names, but I can't find anything user-facing that lets you select channels. It just passes info['ch_names'], then `make_projector` goes through that list and removes the bad channels